docs/zh/deploy/install_deploy.md

```diff
@@ -21,6 +21,8 @@ strings /lib64/libc.so.6 | grep ^GLIBC_
 
 如果需要部署 ZooKeeper 和 TaskManager，则需要有 Java 运行环境。
 
+> Zookeeper 3.4.14需要 `Java 1.7` - `Java 17` 版本。推荐使用 `Java 1.8` 或 `Java 11` 版本。
```

Comment:
https://github.com/4paradigm/OpenMLDB/issues/3766#issuecomment-1955863808
jre 14 and 17failed too, so it should be 'zookeeper can' work on jre >= 14'